roles/business.md

```diff
@@ -2,6 +2,22 @@
 
 ## Business Traits
 
+### +Academy
+
+We need someone:
+
+- to prepare and deliver the weekly learning programmes
+- to provide appropriate feedback and coaching, to help academy members to succeed  
+- to use a variety of teaching methods to challenge and engage academy members
+- to be the PO for the greenfield project
+
+We expect someone in the role:
+
+- to be autonomous in leading the academy
```

Comment:
Since this is a role and not a trait, feels like a scary proposition given we'll be probably bouncing a lot of ideas to keep the academy on course in the first few years.